jme3-core/src/com/jme3/gde/core/util/datatransfer/TransformDataFromOriginal.java

```diff
@@ -0,0 +1,55 @@
+package com.jme3.gde.core.util.datatransfer;
+
+import com.jme3.gde.core.scene.ApplicationLogHandler;
+import com.jme3.gde.core.util.TaggedSpatialFinder;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Node;
+import com.jme3.scene.SceneGraphVisitorAdapter;
+import com.jme3.scene.Spatial;
+
+import java.util.logging.Logger;
+
+/**
+ * Copies Transform data (translation, rotation, scale) from an updated spatial
+ * to the original.
+ */
+public final class TransformDataFromOriginal implements 
```

Comment:
Minor: The name is a bit confusing, as it can be read like "transform this data from original", something like "TransferTransformDataFromOriginal" maybe? "CopyTransformDataFromOriginal"?